tools/maxquant/maxquant.xml

```diff
@@ -1,315 +1,273 @@
-<tool id="maxquant" version="0.1.0" name="MaxQuant">
-  <description></description>
-  <macros>
-    <macro name="input_param">
-      <param format="raw" multiple="true" name="inputs" type="data" label="RAW Inputs" help="" />
-    </macro>
-    <macro name="mod_opts">
-      <options from_data_table="maxquant_mods" />
-      <sanitizer>
-        <valid>
-          <add value="&lt;"/>
-          <add value="&gt;"/>
-          <add value="["/>
-          <add value="]"/>
-        </valid>
-      </sanitizer>
-    </macro>
-    <macro name="protease_opts">
-      <options from_data_table="proteases" />
-    </macro>
-    <macro name="group_params">
-      <param name="protease" label="Enzyme" type="select">
-        <expand macro="protease_opts" />
-      </param>
-      <param name="first_search_tol" label="First Search Tolerance (ppm)" type="float" value="20" />
```

Comment:
Am I reading it correctly that the new wrapper no longer provides a way to interactively set most of the mqpar.xml settings (such as this precursor tolerance)?

Reply:
Yes, currently yes. It would be great to keep the same functionally as before, but on the other hand, this is the first version that really works out of the box. So we could merge that and add more and more options progressively. What do you think?

Reply:
I agree, just wanted to clarify it was intended. Maybe add it to the PR description?